src/site/stages/build/process-cms-exports/transformers/node-page.js

```diff
@@ -42,9 +42,7 @@ function pageTransform(entity) {
     ],
   });
 
-  if (isEmpty(flatten(fieldAlert))) {
-    transformed.fieldAlert = { entity: null };
-  }
+  transformed.fieldAlert = !isEmpty(flatten(fieldAlert)) ? fieldAlert[0] : null;
```

Comment:
Due to the inconsistency you mentioned, are you thinking we'll have to modify the templates to make sure `fieldAlert` doesn't break anything, i.e. `fieldAlert.entity`?

Reply:
Mostly I'm hoping that because `fieldAlert: null` evidently already works, we should be able to just do that consistently. :crossed_fingers: 

Context for anybody reading this without keeping up in the Slacks:
I found that the data we're getting from the GraphQL query isn't consistently returning `fieldAlert: null` or `fieldAlert: { entity: null }`.
![image](https://user-images.githubusercontent.com/12970166/69196241-be4faa00-0ae2-11ea-9a4e-4e233990a6be.png)

This change means that all "empty" `fieldAlert`s will be `null`.